l10n/en/products/vpn/shared.ftl

```diff
@@ -25,11 +25,20 @@ vpn-shared-features-servers = { $servers }+ servers in { $countries }+ countries
 
 vpn-shared-features-bandwidth = No bandwidth restrictions
 vpn-shared-features-activity = No logging of your network activity
+vpn-shared-features-activity-logs = No online activity logs now or ever
+
+# Variables:
+#   $countries (number) - number of available countries
+vpn-shared-features-access-countries = Access to servers in { $countries }+ countries
 
 # Variables:
 #   $devices (number) - maximum number of connected devices
 vpn-shared-features-devices = Option to connect up to { $devices } devices
 
+# Variables:
+#   $devices (number) - number of available devices
+vpn-shared-features-protection = Protection for up to { $devices } devices
```

Comment:
Same as above, this is a very similar sentance to what we have translated already.